src/main/io/gps.c

```diff
@@ -947,6 +1098,26 @@ static void gpsNewData(uint16_t c)
     onGpsNewData();
 }
 
+#ifdef USE_GPS_UBLOX
+ubloxVersion_e ubloxDetectVersion(const char * szBuf, const uint8_t nBufSize) {
+    ubloxVersion_e ret = UBX_VERSION_UNDEF;
+    if (strncmp(szBuf, "00040005", nBufSize) == 0) {
```

Comment:
may be better to use `strtoul()` and then work with binary value (even if temporary copy is necessary)